airflow/jobs.py

```diff
@@ -794,7 +794,7 @@ def create_dag_run(self, dag, session=None):
                 return None
 
             # don't do scheduler catchup for dag's that don't have dag.catchup = True
-            if not dag.catchup:
+            if not dag.catchup and not dag.schedule_interval == '@once':
```

Comment:
Nitpick: Can we use [De Morgan's](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) on this to minimize the number of `not`s?

```python
if not (dag.catchup or dag.schedule_interval == '@once'):
    ...
```

Reply:
Thanks @gwax for the sharp observation. Getting rid of one of the `not`'s would be nice change. @bolkedebruin

Reply:
Will do